File: api/src/main/java/org/pmiops/workbench/actionaudit/ActionType.java
package org.pmiops.workbench.actionaudit;

// do not rename these, as they are serialized as text
public enum ActionType {
  LOGIN,
  LOGOUT,
  BYPASS,
  EDIT,
  CREATE,
  DUPLICATE_FROM,
  DUPLICATE_TO,
  COLLABORATE,
  DELETE,
  DETECT_HIGH_EGRESS_EVENT,
  REMEDIATE_HIGH_EGRESS_EVENT,
  VIEW
}

File: api/src/main/java/org/pmiops/workbench/actionaudit/ActionType.kt
package org.pmiops.workbench.actionaudit

// do not rename these, as they are serialized as text
enum class ActionType {
    LOGIN,
    LOGOUT,
    BYPASS,
    EDIT,
    CREATE,
    DUPLICATE_FROM,
    DUPLICATE_TO,
    COLLABORATE,
    DELETE,
    HIGH_EGRESS,
    COMMENT
}
